### ISSUE TITLE
Downloading hangs on command prompt

### ISSUE THREAD

**Issue by waqarg2001** [udacimak/udacimak#202]
When I render the course, it starts downloading few files. When it reaches to 4 or 5th lesson of first module, the download stops/ha ngs on 98%? Why does that happen? Any solution to it?

**Comment by kendreaditya** [comment]
You have to go to the udaicty root dir via `npm root -g` and inside the udacimak folder you have to upgrade the `youtube-dl` library. I don't suggest you upgrade to the latest `youtube-dl` cause it has some problems, but you can install v3.4.0 via `npm install youtube-dl@3.4.0` (make sure you're in the udacimak folder).

**Comment by nihithreddy** [comment]
@waqarg2001 Did this resolve your issue?

**Comment by waqarg2001** [comment]
@nihithreddy yes it did. But many videos are now downloaded at a very low quality

**Comment by nihithreddy** [comment]
> @nihithreddy yes it did. But many videos are now downloaded at a very low quality

Ok @waqarg2001
